test/algorithms/set_operations/intersection/test_intersection_linear_linear.hpp

```diff
@@ -39,11 +39,11 @@ inline void check_result(Geometry1 const& geometry1,
     BOOST_CHECK_MESSAGE( equals::apply(mls_int1, mls_output, tolerance)
                          || equals::apply(mls_int2, mls_output, tolerance),
                          "case id: " << case_id
-                         << ", intersection L/L: " << bg::wkt(geometry1)
-                         << " " << bg::wkt(geometry2)
-                         << " -> Expected: " << bg::wkt(mls_int1)
-                         << " or: " << bg::wkt(mls_int2)
-                         << " computed: " << bg::wkt(mls_output) );
+                         << " Expected: len=" << bg::length(mls_int1) << " count=" << bg::num_points(mls_int1)
+                         << " or: len=" << bg::length(mls_int2) << " count=" << bg::num_points(mls_int2)
+                         << " Detected: len=" << bg::length(mls_output) << " count=" << bg::num_points(mls_output)
+                         << " wkt=" << bg::wkt(mls_output)
```

Comment:
In general, outputting wkt's in test failures is inconvenient.
Some tests still do it, but it's often unreadable and therefore quite useless.